static-assets/modules/editors/tinymce/v5/tinymce/plugins/acecode/plugin.js

```diff
@@ -65,7 +65,9 @@
 				var dialog = document.getElementsByClassName('tox-dialog--width-lg')[0];
 				dialog.classList.add('fullscreen');
 
-				mce_editor = ace.edit('mce-ace-editor-block');
+				mce_editor = ace.edit('mce-ace-editor-block', {
+				  wrap: true
+        });
```

Comment:
@russdanner @sumerjabri I think this is a preference and I'm not sure which should be the default.
Some people don't like wraps and looks like we're changing behaviour that's been there for a long time.

We have some ace configuration in some an xml (don't recall which at the moment). Shall we add this one there?

Reply:
I think we add the option/config anyway.

@russdanner I'm in favor of the default being `wrap` because that's my personal preference, however, it's your call what should the default be.

I would have loved to see a button on ACE letting me choose in real-time.